packages/flutter/lib/src/widgets/container.dart

```diff
@@ -105,6 +109,54 @@ class DecoratedBox extends SingleChildRenderObjectWidget {
   }
 }
 
+class _DynamicBox extends StatelessWidget {
+  /// This class determines which widget to build based on config's type.
+  const _DynamicBox(this.config, {required this.child});
+
+  final dynamic config;
```

Comment:
Please don't use dynamic. It effectively turns off the type system and makes it easier to introduce bugs.

Reply:
Okay, thanks for letting me know. I got rid of the `_DynamicBox` class to make the PR easier.